src/app/dataset/dataset-list/dataset-list.component.ts

```diff
@@ -24,17 +24,11 @@ export class DatasetListComponent implements OnInit {
               private authenticationBasicService: AuthenticationBasicService) { }
 
   ngOnInit(): void {
-    this.datasetService.getAll({size: this.pageSize, sort: this.sorting}).subscribe(
```

Comment:
Leaving this line as:
`this.datasetService.getAll({size: this.pageSize, sort: this.sorting, params: [{key: 'page', value: 0}]}).subscribe(`
Seems to solve the problem.